packages/fxa-settings/src/lib/integrations/integration-factory.ts

```diff
@@ -183,8 +183,8 @@ export class IntegrationFactory {
     // Important!
     // FxDesktop declares both `entryPoint` (capital P) and
     // `entrypoint` (lowcase p). Normalize to `entrypoint`.
-    const entryPoint = integration.data.getModelData().get('entryPoint');
-    const entrypoint = integration.data.getModelData().get('entrypoint');
+    const entryPoint = integration.data.getModelData('entryPoint');
+    const entrypoint = integration.data.getModelData('entrypoint');
```

Comment:
Just so I'm clear, when should we use `getModelData` vs referring to `integration.data` directly?

Reply:
We can still use `intergration.data` directly; however, we can't get direct access to its data store anymore.

The reason behind this is that we now track if there are changes to the model's data source via an `isDirty` flag. The `isDirty` flag let's us avoid unnecessary model validations. The only way to manage this flag is to encapsulate the object that stores the data so we know when there are changes to it. I'll leave a call out on this change so that this comment makes a bit more sense.

Essentially `integration.data` has two new methods. `getModelData(key)` and `setModelData(key,val)`, and `getModelData()` which provided direct access to the model's data store was removed.